src/include/job.h

```diff
@@ -570,6 +570,7 @@ struct job {
 	int		ji_parent2child_moms_status_pipe;	/* write pipe for parent mom to send sister moms status to child starter process */
 	int		ji_updated;	/* set to 1 if job's node assignment was updated */
 	time_t		ji_walltime_stamp;	/* time stamp for accumulating walltime */
+	struct work_task *ji_bg_hook_task;
```

Comment:
Since this a new pjob field, add to job_alloc() in job_func.c:
#if defined(PBS_MOM) && defined(WIN32)
        pj->ji_momsubt = NULL;
+     pj->ji_bg_hook_task = NULL;           <-- initialize here
#endif